### ISSUE TITLE
[proposal] implement "penalty" when  pub.dev used for political content

### ISSUE THREAD

**Issue by iapicca** [dart-lang/pub-dev#7691]
### use case

I think politics does not belong on pub.dev

<img width="846" alt="Screenshot 2024-05-01 at 19 18 22" src="https://github.com/dart-lang/pub-dev/assets/45637986/3d31acce-e797-4569-a05d-f4d2e16d50fe">

### proposal

I don't think removing the package would be fair and likely would cause issue for the adopters,
but making the content "unsearchable" and only reachable by "exact url" 
eg. `https://pub.dev/packages/penalty_box/easy_localization`
([penalty box](https://en.wikipedia.org/wiki/Penalty_box) like in hockey)

**Comment by iapicca** [comment]
cc @aissat

**Comment by YoussefLasheen** [comment]
I agree. This should also include all pro-ukraine plugs.

**Comment by movaid7** [comment]
For reference, the below packages have had political content in their READMEs for months.

https://pub.dev/packages/flutter_chat_ui
https://pub.dev/packages/flutter_sound
https://pub.dev/packages/flutter_link_previewer

**Comment by bernaferrari** [comment]
I understand the reason but I think it is super hard because you expect someone to be the cop around, which takes time. Will that person be from the community or from Google?  Google itself is extremely super super bad at policing (check the play store horror situation - right now they are even requiring new developer accounts to test the app in TWENTY DEVICES before publishing an app. They really hate developers and don't want them there). Will there be a way to appeal, to be like "I removed, re-list my package"? Ultimately, I think if you don't like, penalize the developer by not using it.

**Comment by aumb** [comment]
@iapicca since you seem to have a strong opinion on this, you should probably fork the packages that annoy you and maintain them yourself, otherwise do not complain about what people do on their OWN packages that THEY maintain. This does not and should not concern pub.dev.

**Comment by samarsheikh001** [comment]
GitHub itself is political, what's next move report GitHub to who??

https://github.blog/2022-03-02-our-response-to-the-war-in-ukraine/

Why boomers only triggered and cry political content when it's Palestine but not Ukraine. Kids are dying let people support whatever they want. If it bothers you that much just skip past it boomer.

**Comment by MaherSafadii** [comment]
Hell no, there is a genocide happening, a second holocaust here in Palestine, nothing is more important in this moment more than the lives of the innocent civilians being slaughtered in Palestine, in addition, there was no one complaining about the Ukrainian flags that storms websites everywhere that lasted for years, FREE PALESTINE

**Comment by dickermoshe** [comment]
The dart team has enough work on their plate maintaining a computer language.
I think we all rather more time be spent on the actual language, adding great features, fixing bugs, than having a senior software engineers play political moderator.

This issue should be ignored by everyone

**Comment by flippin-bits** [comment]
this issue should be closed with no action and this bozo [iapicca](https://github.com/iapicca) should have his account deleted for inciting unnecessary political discourse. There is enough media suppression of the truth, no need to tolerate trolls trying to do the same here.

**Comment by flippin-bits** [comment]
> For reference, the below packages have had political content in their READMEs for months.
> 
> https://pub.dev/packages/flutter_chat_ui https://pub.dev/packages/flutter_sound https://pub.dev/packages/flutter_link_previewer

Yes, see also https://registry.terraform.io/modules/terraform-aws-modules/eks/aws/latest

**Comment by VentGrey** [comment]
Agreed. Code should be the primary focus. Other movements can have their own websites/platforms.

**Comment by VentGrey** [comment]
> Why now? There are many similar repositories, projects, or packages with banners like this, but no one points out anything like this. This is called hypocrisy

Let's open one issue per existing repository then. I stand by “code is everything”, not by the cheap dollar-store morals of GitHub liblefts.

**Comment by rufw91** [comment]
This is hypocrisy plain as day.

**Comment by MeYo0o** [comment]
wow dude , i mean what were you thinking when you posted that issue ?
were you living under a rock since the Russian Ukrainian war ?
there are hundreds of packages that support Ukraine which actually gets all the help she needs from US , Europe 
but for the poor kids and women bombed day and night , suddenly you opened up an issue shortly after Luke got banned because of his garbage tweet regarding that matter , i mean , you don't have to show sympathy for the week , but for the Love of GOD don't be a hypocrite , and when you are asked about this you simply say : "i don't support politics" , well ... those are people and this is Humanity , if you ever heard of it.

**Comment by aytunch** [comment]
What is next? banning pronouns used in peoples profiles? Or their avatars of BLM, Ukraine/Russia/Palestine/Israel/China flag, Rainbow? What is the line we draw to decide what is political or not? I personally care if the package is well maintained and if it is up to coding standards or not. I suggest you ignore stuff which are unrelated to coding and try not to be offended by peoples opinions. It is kind of funny how people started raising concerns lately whereelse we have been seeing such stuff for the past 5 years. Just one example I remember of: https://pub.dev/packages/dismissible_page/versions/0.7.0

**Comment by athomas** [comment]
I've locked the issue for now several comments here are unacceptable, please respect the [code of conduct](https://dart.dev/community/code-of-conduct).
